nil/services/journald_forwarder/main.go

```diff
@@ -148,74 +146,75 @@ func storeData(ctx context.Context, logger logging.Logger, connect driver.Conn,
 		strValue, ok := value.(string)
 		if !ok {
 			logger.Error().Msg("Timestamp is not a string")
```

Comment:
You can write actual value. And it will help during issue debugging.

Reply:
Oh, thanks! Will fix it

>initial idea
Don't miss logs if they wasn't handle for example when clickhouse for some reason wasn't available.

>we limit amount of retries somehow
yes, it's limiting inside otelcol-contrib config. We can setup time for send repeating and batch amount of logs that we can loose when the time expire. Now it's set to unlimited.

> who is responsible to fix this schema
We are :) it's better to understand that we have a problem and fix it than start loosing some logs.